CMakeLists.txt

```diff
@@ -1,17 +1,41 @@
 cmake_minimum_required(VERSION 3.0.2)
 
-project(mxnet C CXX)
+
+
+if((${CMAKE_VERSION} VERSION_GREATER "3.9.0") OR (${CMAKE_VERSION} VERSION_EQUAL "3.9.0"))
+    Set(FIRST_CUDA TRUE)
+else()
```

Comment:
nit: “set” is usually lowercase in this file

Reply:
ok